rfc/2022/2022-08-23_CIT_hostnames.md

```diff
@@ -0,0 +1,246 @@
+
+With the exception of the authentication callbacks that will need to change to the new, environment-specific *platform-api.va.gov hostnames, all settings that required changes have been updated and tested with the help of affected teams. See **Release Steps** for details on updating authentication callbacks.
+
+**Side Note**: In some cases, vets-api code makes subsequent Lighthouse API requests to iterate over paginated data. It is important to ensure that vets-api makes these additional request using the same API URL defined in the settings file (i.e. the forward proxy address and port) rather than directly using the API links provided as part of the API response. This might require some code changes in vets-api that swap out the host portion of the API address to the one defined in the settings.
+
+
+#### **Vets-api: Synthetic Tests**
+Datadog synthetics were added to test a selection of updated host address settings in vets-api.
+
+Example: The `/facilities_api/v1/va` endpoint in vets-api is used by the va.gov Facility Locator app to make an upstream request to the Lighthouse Facilities API. We are rerouting the request to use the forward proxy instead of the api.va.gov hostname. The Datadog test confirms that the vets-api endpoint returns a success response and contains the expected data elements after rerouting the upstream request via the forward proxy. (See: [Rerouting upstream requests from vets-api](https://vfs.atlassian.net/wiki/spaces/ECP/pages/2220785993/2022-06-06+Reroute+vets-api+upstream+requests+to+Lighthouse).)
+
+Reference: [Complete list of Datadog Tests](https://vagov.ddog-gov.com/synthetics/tests?query=tag%3A%28%22team%3Acit%22%29)
+
+
+#### **Vets-website: Change \*api.va.gov References in Unit and Integration Tests**
+We have updated `api.va.gov` references to `platform-api.va.gov` in the test files of the vets-website repository. These updates to the test references did not break any unit or integration tests, as they were in many cases part of canned API responses and not used in any test assertions.
+
+Certain test references to `sandbox-api.va.gov` and `api.va.gov` did not require any changes because these reference endpoints will still be served by the Lighthouse API. **Example**: `health-care-questionnaire` application (not used in Production.)
+
+Parameterizing the API URL was done where possible. Setting the API_URL constant as an environment variable will ensure that unit specs that check for the variable value will pass before and after changing the configured API URL to the new *platform-ap.va.gov hostname.
+
+To prevent the future use of *api.va.gov in test files, the Release Tools Team should add custom linting rules that instruct users to use the new API URLs.
+&nbsp;
+&nbsp;
+#### **Diagram: New Production State with vets-api in BRD and Lighthouse/Kong**
+
+![Prod-release-state-BRD-Kong](https://user-images.githubusercontent.com/1426601/187307838-1ba75418-49e9-479c-8c85-89e8117751eb.png)
```

Comment:
Is there an editable source for these diagrams?

Reply:
Updated arch and release step diagrams to use SVG.